app/organization/members/edit/template.hbs

```diff
@@ -1,61 +1,57 @@
-{{partial 'organization/members-header'}}
+<div class="row">
+  <div class="col-xs-6">
 
-<div class="layout-container">
-  <div class="row">
-    <div class="col-xs-6">
-
-      <div class="panel">
-        <div class="panel-heading">
-          <h3>Edit {{model.name}} roles at {{organization.name}}</h3>
-        </div>
-
-        <div class="panel-body">
-          <div class="form-group">
-            <form role='form'>
-              <label>Roles</label>
-              <div class="form-container">
-                <ul>
-                  {{#each organization.roles as |role|}}
-                    <li class="role">
-                      {{organization-membership-checkbox
-                        user=model
-                        organizationRole=role
-                        changeset=changeset}}
-                      <label>{{role.name}}</label>
-                    </li>
-                  {{/each}}
-                </ul>
-              </div>
-            </form>
-          </div>
+    <div class="panel">
+      <div class="panel-heading">
+        <h3>Edit {{model.name}} roles at {{organization.name}}</h3>
+      </div>
 
-          {{#unless (eq session.currentUser model)}}
-            <div class="">
-              <label>Remove {{model.name}} from {{organization.name}}</label>
-              {{#if disableRemoveMessage}}
-                {{#bs-tooltip title=disableRemoveMessage placement="bottom" bs-contaner=false}}
-                  <div>
-                    <button class='btn btn-danger btn-block' disabled>Remove from {{organization.name}}
-                    </button>
-                  </div>
-                {{/bs-tooltip}}
-              {{else}}
-                <button {{action 'remove' model}} class='btn btn-danger btn-block'>
-                  Remove from {{organization.name}}
-                </button>
-              {{/if}}
+      <div class="panel-body">
+        <div class="form-group">
+          <form role='form'>
+            <label>Roles</label>
+            <div class="form-container">
+              <ul>
+                {{#each organization.roles as |role|}}
+                  <li class="role">
+                    {{organization-membership-checkbox
+                      user=model
+                      organizationRole=role
+                      changeset=changeset}}
+                    <label>{{role.name}}</label>
+                  </li>
+                {{/each}}
+              </ul>
             </div>
-          {{/unless}}
+          </form>
         </div>
```

Comment:
While we're in here, can you expand the form tag to also include the save/cancel buttons?